src/App.js

```diff
@@ -1,24 +1,29 @@
-import logo from './logo.svg';
 import './App.css';
+import Home from './views/Home/Home';
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
+import { getUser } from './services/fetchutils'; 
+import { useState } from 'react';
+import Auth from './views/Auth/Auth';
+import Header from './components/Header';
+
 
 function App() {
+  const [currentUser, setCurrentUser] = useState(getUser());
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <Header currentUser={currentUser} setCurrentUser={setCurrentUser} />
+        <Switch>
+          <Route exact path = '/'>
+            {currentUser ? <Home /> : <Redirect to='/auth'/>}
+          </Route>
+
+          <Route exact path = '/auth'>
+            <Auth setCurrentUser={setCurrentUser}/>
+          </Route>
```

Comment:
Here we also want to check for a current user so a signed in user cannot access the auth page